modules/olm-operatorgroups-limitations.adoc

```diff
@@ -1,12 +1,12 @@
-[id="olm-operatorgroups-limitations"]
-= Limitations for multi-tenant Operator management
+// Module included in the following assemblies:
+//
+// * operators/understanding/olm/olm-understanding-operatorgroups.adoc
 
-{product-title} provides limited support for simultaneously installing different variations of an Operator on a cluster. Operators are control plane extensions. All tenants, or namespaces, share the same control plane of a cluster. Therefore, tenants in a multi-tenant environment also have to share Operators.
+[id="olm-operatorgroups-limitations"]
+= Limitations for multitenant Operator management
 
-The Operator Lifecycle Manager (OLM) installs Operators multiple times in different namespaces. One constraint of this is that the Operator’s API versions must be the same.
+{product-title} provides limited support for simultaneously installing different variations of an Operator on a cluster. Operators are control plane extensions. All tenants, or namespaces, share the same control plane of a cluster. Therefore, tenants in a multitenant environment must also share Operators.
 
-Different major versions of an Operator often have incompatible custom resource definitions (CRDs). This makes it difficult to quickly verify OLMs.
```

Comment:
I found this "Limitations for multitenant Operator management" section in the OperatorGroups topic that I didn't realize was added a while back. I've cleaned it up a little and it seems like it likely can stay (although it now repeats a bit of what is discussed in the newer sections), but I wasn't sure what was originally meant by "verify OLMs" on this line.

For now, I've just dropped it vaguely to "difficult to quickly verify." until there's a better suggestion.